RNTester/Podfile.lock

```diff
@@ -348,34 +348,34 @@ EXTERNAL SOURCES:
   Folly: feff29ba9d0b7c2e4f793a94942831d6cc5bbad7
   glog: b3f6d74f3e2d33396addc0ee724d2b2b79fc3e00
-  RCTRequired: fa143a98ccd5f5c952f07364a331c75b8dc4e663
-  RCTTypeSafety: 1d976d01072f9968ab3c2cafb2eb7f6ca7c48fdd
-  React: 73d4569aed422237bfb5ec1f471aa4d8c045b331
-  React-ART: 07fa82267a87e67e0c3523a5f595e86504658d0b
-  React-Core: 277537483a5b0c41beae832d1c7a9794561b6529
-  React-CoreModules: 4e9e1c821a7a5fafb3ffd9f9ba085b40f477da83
-  React-cxxreact: 22e64325458325149e51111832d1005fcf27c0b5
-  React-jsi: 9d087842253c8d0775b318caeccbdee547c89316
-  React-jsiexecutor: f0043f9030ec0c79b937fd6ab8f7c4f06a7be241
-  React-jsinspector: 503fb0b052442aed724f735789174754b2b72687
-  React-RCTActionSheet: 82b3af83ed5043659bc39316966571e4b1b7ef4e
-  React-RCTAnimation: a48f680c9c5e55c27bf945d5864eb5d638da1576
-  React-RCTBlob: 9c266ac30bfbfb10bb62b3f74c049febf80fc2e5
-  React-RCTImage: 5889e573cea7e140a447d4cd4658c87885669d81
-  React-RCTLinking: 4699cfc4eba902b7f7f1652178af4d8cc18ee971
-  React-RCTNetwork: 5fea7fcf7bbe49a37dfa9fddc499dff6506297b7
-  React-RCTPushNotification: aba4d4cb1f0f851b2df81b11cb3c402dd3123cf0
-  React-RCTSettings: ec7ee3a25d2f16a69936335bf7ef8f9d61c70fb0
-  React-RCTTest: ed828653bca00aa9f5071f554b54b4d36a254648
-  React-RCTText: d22b5f61ed6db5670e6aaedc183337b6f7c73337
-  React-RCTVibration: 80c7b79af833447615624f16cf1ec6bed8babccc
-  ReactCommon: 10f21f50611c651a89b8323520ca71eac9f7df2b
-  Yoga: f18826821d9110f5d57c15dadabe63ba8b75c2e2
+  RCTRequired: 622690f1bcbe26868f52961e719d860f3004b6b6
+  RCTTypeSafety: 3ea632ebbb03638caff9e84f6f6abb7e936a3e5b
+  React: 8d5eec1276f11a81b19d8abb6443dcc39a110300
+  React-ART: c013918bd3d8c62b8ad582031fd9a760c6df53d8
+  React-Core: bd781c39a56a76af11a6f9657c5aaff3bbff46ff
+  React-CoreModules: de492a564074fb323f5a7577cc8fad169f6b35a1
+  React-cxxreact: 9797c881683a4d7cb3755a533942745f0244fc0b
+  React-jsi: 30ffe0e2af47cf815eb7ee61084ea891cbf48cbb
+  React-jsiexecutor: f1080a4b5aa627a393f8b9e2b545f98e32b1e8ed
+  React-jsinspector: 5e2444bdfaf148c9f14b9ed68dd607cd12a3c201
+  React-RCTActionSheet: 96944bdeca027606bdb9c798b61d82ee34c12105
+  React-RCTAnimation: 14a267ca1675e8aa46038e54cedaa568089ca72f
+  React-RCTBlob: 8df5ae4defda691fa058b42b1d2bc253ceb8bcd5
+  React-RCTImage: 51693ef8b6539c75a39046569894c46936f9e46f
+  React-RCTLinking: 3925342df1e9de2c4b8674cca39de0e7a42d0ccf
+  React-RCTNetwork: de7a645b95657b88fd891e0b59298b7bf1a50ad9
+  React-RCTPushNotification: 61aff4a48cd87145c170a173426c1bec96b288d3
+  React-RCTSettings: 5c4b741b59a4547f8f002a3c17bc9b817298c8e0
+  React-RCTTest: 5afe9d3c7ab66dfd2ca5145f9189c0a85f2fb34b
+  React-RCTText: 09b6d9d7363064d23380f50fc01af3ab75131cf7
+  React-RCTVibration: 8135beb48e3065fbcbe6417b9a6bbbc884c02a78
+  ReactCommon: a377fdc9faecc5c9408bcd07ce997703c0ce22ff
+  Yoga: 7d784f976888661a2cea9333ab06125b1133e435
 
 PODFILE CHECKSUM: 1a5700c13b2bc8ffc5ed4282d795e8909c883f63
 
-COCOAPODS: 1.9.3
```

Comment:
nit: need to update your cocoapods `brew update cococapods`

Reply:
Alternatively, RNTester comes with a `Gemfile` nowadays, meaning that if you do the following you should always be using the same CP version that others are:

```bash
gem install bundler
bundle install
bundle exec pod install
```